malwarebytes.sls

```diff
@@ -1,30 +1,36 @@
-# just 32-bit x86 installer available
-{% if grains['cpuarch'] == 'AMD64' %}
-    {% set PROGRAM_FILES = "%ProgramFiles(x86)%" %}
-{% else %}
-    {% set PROGRAM_FILES = "%ProgramFiles%" %}
-{% endif %}
 # Source: http://www.malwarebytes.org
+# This is a terrible setup, as Malwarebytes give you a single installer with no version information, nor do they release previous versions to
+# download from their official site. 
+# Use this with caution.
```

Comment:
Chrome does the same thing. They do not provide previous versions, only an MSI of the latest version. The way we handle this scenario is to use `latest` as the version as we have no way of knowing what the actual version is. For example, look at the Software Definition for Chrome:
https://github.com/saltstack/salt-winrepo-ng/blob/master/chrome.sls